packages/platform-server/src/server.ts

```diff
@@ -135,3 +139,37 @@ export function platformServer(extraProviders?: StaticProvider[] | undefined): P
 
   return platform;
 }
+
+/**
+ * Bootstraps an instance of an Angular application for the server platform.
+ *
+ * @usageNotes
+ * The root component passed into this function *must* be a standalone.
+ *
+ * ```ts
+ * @Component({
+ *   template: 'Hello world!'
+ * })
+ * class RootComponent {}
+ *
+ * const appRef: ApplicationRef = await bootstrapServerApplication(RootComponent, {
+ *   providers: [
+ *     {provide: BACKEND_URL, useValue: 'https://yourdomain.com/api'}
+ *   ]
+ * });
+ * ```
+ *
+ * @param rootComponent A reference to a standalone component that should be rendered.
+ * @param options Extra configuration for the bootstrap operation, see `ApplicationConfig` for
+ *     additional info.
+ * @returns A function that returns a promise that returns an `ApplicationRef` instance once resolved.
+ *
+ * @publicApi
+ */
+export function bootstrapServerApplication(
+  rootComponent: Type<unknown>,
+  options: ApplicationConfig,
+): (platformInjector: Injector) => Promise<ApplicationRef> {
+  return (platformInjector: Injector) =>
+    bootstrapApplication(rootComponent, options, platformInjector);
```

Comment:
I'm wondering if instead of updating the public API of the `bootstrapApplication`, we can use an internal version of it with an extra argument?
```suggestion
    bootstrapApplicationInternal(rootComponent, options, platformInjector);
```

Reply:
I have create a new method `bootstrapApplicationInternal `.